src/debuggee.ts

```diff
@@ -75,9 +79,9 @@ export class Debuggee {
    *     that the user has a way of noticing.
    *     TODO(ofrobots): has this been renamed to `status` in the API?
    */
-  constructor(properties: DebuggeeProperties) {
+  constructor(properties: DebuggeeProperties, appInfo: AppInfo) {
```

Comment:
can we make appInfo one property in DebuggerProperties?

Reply:
Good idea.  Done.